package.json

```diff
@@ -2,7 +2,10 @@
   "name": "sars-cov-2-docs",
   "version": "1.0.0",
   "description": "",
-  "dependencies": {},
+  "dependencies": {
+    "@observablehq/runtime": "^4.7.0",
+    "@stevenweaver/case-vs-sequence-count": "https://api.observablehq.com/@stevenweaver/case-vs-sequence-count.tgz?v=3"
```

Comment:
How does the versioning work here; that is, when you edit the root observable do we have to bump this `3` each time?